plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiMetadata.java

```diff
@@ -246,6 +246,13 @@ public List<SchemaTableName> listTables(ConnectorSession session, Optional<Strin
         return tableNames.build();
     }
 
+    @Override
+    public void renameTable(ConnectorSession session, ConnectorTableHandle tableHandle, SchemaTableName newTableName)
```

Comment:
I don't think adding support for renaming tables without supporting write operations is a good idea from test coverage perspective.

Reply:
@ebyhr  I have a use case where I need to replace an existing Hudi table with a Trino view that points to an Iceberg table as part of a migration process. To achieve this, I first need the ability to rename the Hudi table.